Changelog.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 ## [unreleased]
+- Add display name to settings (#4937)
+- Add settings page (#4922)
```

Comment:
Delete this line; when I merged in the other one, I moved this entry further down. In fact, I'd like to keep these entries in roughly PR numerical order, so please move the "Add display name to settings" further down as well.